src/Raven.Studio/typescript/commands/maintenance/getAdminLogsSettingsCommand.ts

```diff
@@ -0,0 +1,14 @@
+import commandBase = require("commands/commandBase");
+import endpoints = require("endpoints");
+
+class getAdminLogsSettingsCommand extends commandBase {
```

Comment:
can we rename to getAdminLogsConfiguration to match url, object and error message?

Reply:
done